src/site/twitch.tv/modules/chat/ChatModule.vue

```diff
@@ -390,7 +390,17 @@ export const config = [
 			gridMode: "new-row",
 		},
 		label: "Custom Highlights",
-		hint: "Create custom highlights for specific words or phrases",
+		hint: "Create custom highlights",
+		defaultValue: new Map(),
+	}),
+	declareConfig<Map<string, IgnoreDef>>("chat.ignores", "CUSTOM", {
+		path: ["Chat", "Ignores"],
+		custom: {
+			component: markRaw(SettingsConfigHighlights),
```

Comment:
@AnatoleAM how can I pass a prop to this?

Reply:
You may need to add a new property there, `componentProps`, which you could then set to `v-bind` at `SettingsNode.vue:19`

Reply:
Alternatively you could wrap it under another component, and set the prop there.